requirements.txt

```diff
@@ -1,5 +1,5 @@
 # remember to also update unittest_requirements.txt when updating this file
-pulpcore>=3.20.0.dev
+pulpcore>=3.20.0.dev,<3.25
```

Comment:
I have introduced this change to make the CI green again. 3.25 will contain breaking changes. Having an upper bound makes sense to me.